app.rb

```diff
@@ -78,7 +78,7 @@ def list_rentals_for_id
     person_id = gets.chomp.to_i
     puts 'Rentals: '
     @rentals.each do |rental|
-      if rental.person.id == person_id
+      if rental.person && rental.person.id == person_id
```

Comment:
- [ ] @arabiu033 and @AdnanOlarmmi great work on your project however all requests for input and input validation should've been placed in the main.rb file and/or classes/methods whose responsibilities are to request or validate input in your case either the main or logic file this is in order to achieve the 'Single-responsibility Principle'.

Reply:
Alright.
@AdnanOlarmmi Thanks for the feedback